UiPath.FreeRdpClient/UiPath.FreeRdpClient/Internals/Logging.cs

```diff
@@ -66,24 +69,36 @@ private void RegisterThreadScope(string scope)
     private bool FilterLogs(LogLevel logLevel, string message)
     {
         if (logLevel is LogLevel.Error
-            && FilterNotStartsWith.Any(message.StartsWith))
+            && FilterRemoveStartsWith.Any(message.StartsWith))
             return false;
 
         return true;
     }
 
-    Task IHostedService.StartAsync(CancellationToken cancellationToken)
+    private void EnableNativeLogsForwarding()
     {
         var forwardFreeRdpLogs = Environment.GetEnvironmentVariable("WLOG_FILEAPPENDER_OUTPUT_FILE_PATH") is null;
-        NativeInterface.InitializeLogging(logCallback: _logCallbackDelegate,
+        NativeInterface.InitializeLogging(logCallback: LogCallbackDelegate,
                                           registerThreadScopeCallback: _registerThreadScopeCallbackDelegate,
                                           forwardFreeRdpLogs: forwardFreeRdpLogs);
-        return Task.CompletedTask;
+        _logsForwardingEnabled = true;
     }
 
-    Task IHostedService.StopAsync(CancellationToken cancellationToken)
+    private void DisableNativeLogsForwarding()
     {
         LoggerFactory = null;
-        return Task.CompletedTask;
+        NativeInterface.InitializeLogging(logCallback: null, registerThreadScopeCallback: null, forwardFreeRdpLogs: false);
+        _logsForwardingEnabled = false;
+    }
+
+    internal void EnsureNativeLogsForwarding()
```

Comment:
As time the EnableNativeLogsForwarding is done in constructor, EnsureNativeLogsForwarding is not needed (also _logsForwardingEnabled not needed)